modules/local/run_helixfold3/Dockerfile

```diff
@@ -1,16 +1,16 @@
 ARG CUDA=12.8.1
-FROM nvidia/cuda:${CUDA}-cudnn-devel-ubuntu22.04 AS builder
+FROM nvidia/cuda:${CUDA}-cudnn-devel-ubuntu22.04
```

Comment:
Why are we getting read of the multi-stage build, was not this a way of shrinking image size?